docs/UserGuide.md

```diff
@@ -144,6 +144,7 @@ In this section, you will learn about the commands available in Pawbook and how
 **Function**: Adds a **dog/owner/program** to Pawbook.<br>
 **Usage**: When you want to add an entity into Pawbook, you can simply use the add command. The add command adds the new entity at the end of the current list. 
 For instance, when a new dog arrives at the school with its owner, you will need to add the owner profile and the dog profile and store it in Pawbook. 
+Note that the date-time format used for sessions is d-M-yyyy HH:mm.
```

Comment:
As in I think that even this format it will be good to specify what each one of them means, like mm refers to minutes should be 00-59 etc, to prevent some lunatics from going crazy. Also this portion is kind of general, you might want to explicitly mention that the birth date of the dogs follows a similar format, omitting the time portion and that the date has to be on or before the current date.